lhapdf.sh

```
@@ -20,15 +21,7 @@ prefer_system_check: |
 ---
 #!/bin/bash -ex
-case $ARCHITECTURE in
-  osx*)
-    # If we preferred system tools, we need to make sure we can pick them up.
-    [[ ! $AUTOTOOLS_ROOT ]] && PATH=$PATH:$(brew --prefix gettext)/bin
-  ;;
-  *)
-    EXTRA_LD_FLAGS="-Wl,--no-as-needed"
-  ;;
-esac
+EXTRA_LD_FLAGS="-Wl,--no-as-needed"
```

Shellcheck (0.11.0)

[warning] EXTRA_LD_FLAGS appears unused. Verify use (or export if used externally).

(SC2034)